presto-tests/src/test/java/com/facebook/presto/tests/TestSqlFunctions.java

```diff
@@ -267,4 +267,20 @@ public void testShowCreateFunctions()
         assertQueryFails("SHOW CREATE FUNCTION array_agg", "SHOW CREATE FUNCTION is only supported for SQL functions");
         assertQueryFails("SHOW CREATE FUNCTION presto.default.array_agg", "SHOW CREATE FUNCTION is only supported for SQL functions");
     }
+
+    @Test
+    void testParameterCaseInsensitive()
+    {
+        @Language("SQL") String createFunctionInt = "CREATE FUNCTION testing.common.array_append(input array<int>, x int)\n" +
+                "RETURNS array<int>\n" +
+                "RETURN concat(inPut, array[x])";
+        @Language("SQL") String createFunctionDouble = "CREATE FUNCTION testing.common.array_append(inPut array<bigint>, x bigint)\n" +
+                "RETURNS array<bigint>\n" +
+                "RETURN concat(input, array[x])";
+        assertQuerySucceeds(createFunctionInt);
+        assertQuerySucceeds(createFunctionDouble);
+
+        assertQuery("SELECT testing.common.array_append(array[1, 2, 3], 4)", "VALUES array[1, 2, 3, 4]");
+        assertQuery("SELECT testing.common.array_append(array[bigint'1', bigint'2', bigint'3'], bigint'4')", "VALUES array[1L, 2L, 3L, 4L]");
+    }
```

Comment:
May be one more test for using non-params (like columns outside) and also the function name case insensitively like:

SELECT array_APPEND(Array, iTem) from (SELECT array[1] array, 2 ITEM)